scripts/o3de/ExportScripts/export_source_android.py

```diff
@@ -0,0 +1,383 @@
+
+    exp.process_command([o3de_cli_script_path, 'android-generate', '-p', project_name, '-B', target_android_project_path, "--asset-mode", asset_pack_mode], 
+                cwd=default_base_path)
+
+    exp.process_command([target_android_project_path / ('gradlew'+ (".bat" if is_pc else "")), f'assemble{build_config.title()}'], cwd=target_android_project_path)
+
+    if deploy_to_device:
+        deloy_to_android_device(target_android_project_path,
+                                build_config, 
+                                project_name, 
+                                android_sdk_home,
+                                org_name,
+                                activity_name)
+
+def export_source_android_parse_args(o3de_context: exp.O3DEScriptExportContext, export_config: command_utils.O3DEConfig):
+    parser = argparse.ArgumentParser(
+                    prog=f'o3de.py export-project -es {__file__}',
+                    description="Exports a project as standalone to the desired output directory with release layout. "
+                                "In order to use this script, the engine and project must be setup and registered beforehand. ",
+                    epilog=exp.CUSTOM_CMAKE_ARG_HELP_EPILOGUE,
+                    formatter_class=argparse.RawTextHelpFormatter,
+                    add_help=False
+    )
+    parser.add_argument(exp.CUSTOM_SCRIPT_HELP_ARGUMENT,default=False,action='store_true',help='Show this help message and exit.')
+    
+    parser.add_argument('-deploy', '--deploy-to-android',default=False,action='store_true',help='At completion of build, deploy to android device.')
+
+    parser.add_argument('-actname', '--activity_name', type=str, default='',
+                        help='The name of the activity to invoke when deploying and running the APK.')
+    
+    parser.add_argument('-orgname', '--org_name', type=str, default='',
+                        help='The name of the organization identifier (i.e. org.o3de.PROJECT) to use when deploying and running the APK.')
+
+    parser.add_argument('-defaults', '--provide-default-android-config',default=False,action='store_true',help='Sets up a new android configuration if requested.')
+    
+    parser.add_argument('-out', '--android-output-path', type=pathlib.Path, required=True, help='Path that describes the final resulting Android Project and APK location.')
+
+    default_project_build_config = export_config.get_value(key=exp.SETTINGS_PROJECT_BUILD_CONFIG.key,
+                                                            default=exp.SETTINGS_PROJECT_BUILD_CONFIG.default)
+    parser.add_argument('-cfg', '--config', type=str, default=default_project_build_config, choices=[exp.BUILD_CONFIG_RELEASE, exp.BUILD_CONFIG_PROFILE],
+                        help='The CMake build configuration to use when building project binaries.')
+    
+    parser.add_argument('-apkm', '--asset_pack_mode', type=str, default='PAK', choices=['PAK', 'LOOSE'],
```

Comment:
`asset_pack_mode` doesn't make sense. Its PAK or LOOSE asset mode. And lets use the same argument as https://github.com/o3de/o3de/blob/8fc008743313813933cd0be190535fc4ed79a92b/scripts/o3de/o3de/android.py#L579-L584